RedHat-UX/red-hat-design-system
# fix(button): fixing dark theme `danger` variant tokens for accessibility

## What I did

1.  Updated the token values for `dark` mode `danger` variant from `--rh-color-text-primary-on-dark` to `--rh-color-text-primary-on-light` to accessibility


## Testing Instructions

1. Go to Deploy Preview
2. Check to make sure that the text on `danger` button for `dark` mode is `#151515`
